src/brain/issueLabelHandler/followups.ts

```diff
@@ -158,5 +166,23 @@ export async function ensureOneWaitingForLabel({
 
   await modifyProjectIssueField(itemId, labelName, STATUS_FIELD_ID, octokit);
 
+  let timeToTriageBy
```

Comment:
Not the best name since it encompasses both triage, routing, and (soon) stale. Eh?

Reply:
you're right :)

let's change that